packages/synapse-react-client/src/components/ChallengeTeamWizard/ChallengeTeamTable.tsx

```diff
@@ -102,24 +118,39 @@ export default function ChallengeTeamTable(props: ChallengeTeamTableProps) {
         disableColumnMenu: true,
         renderCell: ({ row }) => (
           <Link
-            to={{
-              pathname: `${getEndpoint(
-                BackendDestinationEnum.PORTAL_ENDPOINT,
-              )}/#!Team:${row.id}`,
-            }}
+            href={`${getEndpoint(
+              BackendDestinationEnum.PORTAL_ENDPOINT,
+            )}/#!Team:${row.id}`}
             target="_blank"
+            underline={'hover'}
           >
             {row.name}
           </Link>
         ),
       },
+      {
+        field: 'hasInvitation',
+        headerName: '',
+        width: 15,
+        filterable: false,
+        hideable: false,
+        disableColumnMenu: true,
+        align: 'center',
+        renderCell: ({ value }) =>
+          value ? (
+            <Tooltip title={'You have been invited to join this team'}>
+              <EmailTwoTone fontSize={'small'} sx={{ color: 'grey.700' }} />
+            </Tooltip>
+          ) : null,
+      },
       {
         field: 'created',
         headerName: 'Created On',
         width: 100,
         filterable: false,
         hideable: false,
         disableColumnMenu: true,
+        renderCell: ({ value }) => formatDate(dayjs(value), 'MM/DD/YY'),
```

Comment:
Moved this transformation to the column definition instead of when creating the row objects